llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -393,7 +393,7 @@ X86TargetLowering::X86TargetLowering(const X86TargetMachine &TM,
   }
 
   for (auto Op : {ISD::FP16_TO_FP, ISD::STRICT_FP16_TO_FP, ISD::FP_TO_FP16,
-                  ISD::STRICT_FP_TO_FP16}) {
+                  ISD::STRICT_FP_TO_FP16, ISD::STRICT_FP_TO_BF16}) {
```

Comment:
@phoebewang Do you mean here, we need to drop `Custom` for `Subtarget.hasF16C()` if `Op` is `ISD::STRICT_FP_TO_BF16`? If yes, I will take it out of the loop.

Reply:
Yes.

Reply:
I don't know how to implement the `Custom` path since I'm not familiar with what instructions need to be used. I leave a FIXME there.